src/resizable.directive.ts

```diff
@@ -688,7 +692,10 @@ export class ResizableDirective implements OnInit, OnChanges, OnDestroy {
         };
         const resizeCursors = getResizeCursors();
         const cursor = getResizeCursor(currentResize.edges, resizeCursors);
-        this.renderer.setStyle(document.body, 'cursor', cursor);
+        // browser check for angular universal, because it doesn't know what document is
+        if (isPlatformBrowser(this.platformId)) {
```

Comment:
Is it possible to get the same result by injecting the document in the constructor like so:

```
@Inject(DOCUMENT) private document: any
```

and then you can just do `this.document.body` and universal will mock it out?

Reply:
Also is this actually needed here? It's only called on a mousedown event which can never fire on the server

Reply:
Actually the first version with Object was the right one, but then Travis complained. Any version of Object, object or any works in practice.

Reply:
But I recommend Object cause this is what the isPlatformBrowser() expects.
There is an open ticket about this here 
https://github.com/angular/angular/issues/23904

Reply:
Got it, you can either set the type as `any` or as `Object` and add `// tslint:disable-line ban-types` to workaround tslint complaining

Reply:
It passed with any and would leave it like this, maybe angular decides some day to follow the recommendations and use object and I don‘t want to get the error again.